### PR TITLE
Fix header spacing on Governance tab

### PR DESCRIPTION
**Pull request by jholdstock**
Ensure the spacing between Title and Description is 5px, as specified in #3404. More detail in commit messages.

### CODE REVIEW

**Review by bgptr — CHANGES_REQUESTED** [review]
@jholdstock, thanks for this PR. Looks nice, only the description part looks a bit more crowded now. There should be `10px` margin between the description lines in the latest design: https://www.figma.com/file/EanUbqJmPs8EMqY26j9zwo/decred---piui---decrediton?node-id=92%3A2457

design vs current state:
image

**Comment by jholdstock** [comment]
Right, I see the problem here. I was only seeing one line of description because I had the external HTTP request for treasury balance disabled. Fixing

**Comment by jholdstock** [comment]
Pushed some changes,  I think this now also fixes #3627

File: app/components/layout/TabbedPage/TabbedPage.module.css [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
-  padding: 40px 60px 0 96px;
+  padding: 25px 60px 0 96px;
   position: relative;
   display: flex;
 }
 
 .tabContent {
   background-color: var(--background-container);
-  top: 71px;
+  top: 55px;
```
Comment (jholdstock) [inline app/components/layout/TabbedPage/TabbedPage.module.css:31]: :nauseated_face:

**Review by bgptr — APPROVED** [review]
Tested, and looks nice.

**Review by alexlyp — APPROVED** [review]

**Pull request merged**